proofread: fio.md (#1575)
juicedata/juicefs: docs/en/benchmark/fio.md

# Benchmark with fio

:::tip

> JuiceFS v1.0+ has Trash enabled by default. The benchmark creates and deletes temporary files in the file system, and eventually dumps these files into a directory named `.trash`. To avoid storage space being occupied by `.trash`, you can run command `juicefs config META-URL --trash-days 0` to perform benchmark with Trash disabled. See [trash](../security/trash.md) for details.

Reviewer: ```suggestion
Trash is enabled in JuiceFS v1.0+ by default. As a result, temporary files are created and deleted in the file system during the benchmark, and these files will be eventually dumped into a directory named `.trash`. To avoid storage space being occupied by `.trash`, you can run command `juicefs config META-URL --trash-days 0` to disable Trash before benchmark. See [trash](../security/trash.md) for details.
```

这一段好像是和mdtest重复的？ :P

:::

## Testing Approach

> Performed a sequential read/write benchmark on JuiceFS, [EFS](https://aws.amazon.com/efs) and [S3FS](https://github.com/s3fs-fuse/s3fs-fuse) with [fio](https://github.com/axboe/fio).

Reviewer: ```suggestion
Perform a sequential read/write benchmark on JuiceFS, [EFS](https://aws.amazon.com/efs) and [S3FS](https://github.com/s3fs-fuse/s3fs-fuse) with [fio](https://github.com/axboe/fio).
```
在mdtest里用的是perform，最好统一一下？

## Testing Environment

> The following tests are all performed using `fio` on a c5d.18xlarge EC2 instance (72 CPU, 144G RAM) with Ubuntu 18.04 LTS (Kernel 5.4.0) operating system. JuiceFS use a local Redis instance (version 4.0.9) to store metadata.

Reviewer: ```suggestion
All the following tests are performed using `fio` on a c5d.18xlarge EC2 instance (72 CPU, 144G RAM) with Ubuntu 18.04 LTS (Kernel 5.4.0) operating system. JuiceFS uses the local Redis instance (version 4.0.9) on the same host to store metadata.
```
不知道这里<同主机的本地 Redis>里的<同主机>是不是重点，需不需要强调一下？

Author: 这里 local 就是 on the same host 的意思啦
